lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -136,6 +136,11 @@ void LLVMIRGen::loadBaseAddresses(llvm::IRBuilder<> &builder) {
   offsetsArray_ = F->args().begin() + 3;
 }
 
+static const unsigned char libjit_bc[] = {
```

Comment:
Add a comment here explaining where the content of libjit comes from or how it is generated.